apps/ex_wire/lib/ex_wire.ex

```diff
@@ -26,11 +26,17 @@ defmodule ExWire do
   defp get_children(_params) do
     chain = ExWire.Config.chain()
 
+    perform_discovery = Config.perform_discovery?()
+
     sync_children =
       if Config.perform_sync?() do
+        # If we're not performing discovery, let's immediately connect
```

Comment:
Seems like this is clear for the code below. How would you feel about removing this comment?

Reply:
I mostly added this because geth/parity, I believe, do this, but never really make it clear. I figured it'd be nice to spell this out.